modules/theme/src/theme-core/definitions/apps/developer-portal.less

```
@@ -1011,10 +1011,15 @@
 
     .protocol-card {
         background-color: @white !important;
+        border-color: #e8e8e8 !important;
```

Can't we use the default border color here.
```suggestion
        border-color: @defaultBorderColor !important;
```